freezeyt_blog/app.py

```diff
@@ -1,28 +1,42 @@
 import mistune
 from flask import Flask, url_for, abort, render_template
 from pathlib import Path
+from urllib.parse import urlparse
 
 from pygments import highlight
 from pygments.lexers import get_lexer_by_name
 from pygments.formatters import HtmlFormatter
 
 app = Flask(__name__)
+print(app.config)
 
 BASE_PATH = Path(__file__).parent
 CONTENT_PATH = BASE_PATH / 'content/'
 
 ARTICLES_PATH = CONTENT_PATH / 'articles/'
-STATIC_PATH = CONTENT_PATH / 'images/'
+IMG_PATH = 'static/images/'
 
-class HighlightRenderer(mistune.Renderer):
+app.config['IMAGES_PATH'] = ['static/images']
+
+class BlogRenderer(mistune.Renderer):
     def block_code(self, code, lang):
         if not lang:
-            return '\n<pre><code>%s</code></pre>\n' % \
-                mistune.escape(code)
+            return '\n<pre><code>%s</code></pre>\n' %mistune.escape(code)
+
```

Comment:
`%` je starý způsob formátování řetězců; v moderním Pythonu můžeš použít f-string:
```suggestion
            escaped = mistune.escape(code)
            return f'\n<pre><code>{escaped}</code></pre>\n' 
```